docs/ui-cheatsheet.md

```
@@ -208,6 +208,32 @@ Two layouts share `<WikiView>`: the **index** (page list) and a **single page**
 ```
 
+### page-edit (canvas timeline only — #963)
+
+When the LLM Writes/Edits a `data/wiki/pages/<slug>.md` file via
+Claude Code's built-in tools, the snapshot endpoint publishes a
+synthetic `manageWiki` toolResult with `action: "page-edit"` into
+the active session. The canvas (StackView) renders it via the
+same `<WikiView>` component as `action: "page"`, so the body
+markup is identical:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)